docs/BUILD.bazel

```diff
@@ -9,8 +9,13 @@ _DOCS = ["aspect.md"] + [
 
 genrule(
     name = "command_list_bzl",
+    # We need to be able to start bazel so that we can get the correct flags
+    srcs = [
+        "//:WORKSPACE",
```

Comment:
does it really need all the real file? could a minimal workspace work?

Reply:
Yes that worked